src/controller/TypedReadCallback.h

```diff
@@ -163,6 +163,8 @@ class TypedReadEventCallback final : public app::ReadClient::Callback
         mOnError(nullptr, Protocols::InteractionModel::Status::Failure, aError);
     }
 
+    void OnDeallocatePaths(chip::app::ReadPrepareParams && aReadPrepareParams) override {}
```

Comment:
What about fixing up the existing `SubscribeAttribute` and `SubscribeEvent` APIs to support specifying a re-subscription boolean?

Reply:
Updated, thanks